src/LoggedInUser/LoggedInUser.js

```diff
@@ -2,18 +2,34 @@ import React, { Component } from "react";
 import "./LoggedInUser.css";
 import MoviesContainer from "../MoviesContainer/MoviesContainer";
 import MovieDetailsPage from "../MovieDetailsPage/MovieDetailsPage";
-import { getMovie, postUserMovieRating } from "../apiCalls";
+import { getMovie, getUsersRatings, postUserMovieRating } from "../apiCalls";
 
 class LoggedInUser extends Component {
   constructor(props) {
     super(props);
     this.state = {
       selectedMovie: null,
-      userRatings: []
+      userRatings: [],
+      userID: this.props.loggedInUserData.user.id
     };
     this.url = "https://rancid-tomatillos.herokuapp.com/api/v2";
   }
 
+  componentDidMount = () => {
```

Comment:
This is great - we'll figure out if this is what is needed once router is implemented